src/main/java/Duke.java

```diff
@@ -1,10 +1,193 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    private static void printLine(){
+        System.out.println("---------------------------------------------------");
+    }
+
+    private static String getStringFromList(String[] inputList, int fromIndex, int toIndex) throws IllegalArgumentException{
```

Comment:
Perhaps you can maintain consistency in adding a space before the opening brace of function declarations.